src/components/application_manager/src/message_helper.cc

```diff
@@ -2395,24 +2395,21 @@ mobile_apis::Result::eType MessageHelper::ProcessSoftButtons(
     }
 
     switch (request_soft_buttons[i][strings::type].asInt()) {
-      case SoftButtonType::SBT_IMAGE: {
-        // Any text value for type "IMAGE" should be ignored.
-        if (request_soft_buttons[i].keyExists(strings::text)) {
-          request_soft_buttons[i].erase(strings::text);
-        }
-
-        if ((!request_soft_buttons[i].keyExists(strings::image) ||
-             (Result::SUCCESS !=
-              VerifyImage(
-                  request_soft_buttons[i][strings::image], app, app_mngr)))) {
-          return Result::INVALID_DATA;
-        }
-        break;
-      }
+      case SoftButtonType::SBT_IMAGE:
       case SoftButtonType::SBT_TEXT: {
```

Comment:
@indlu, are you intentionally leave `SBT_IMAGE` to be processed with `SBT_TEXT`?
It's unclear why this two should be processed as one in any case. Can you write a comment on it?